src/main/kotlin/io/github/inductiveautomation/kindling/idb/tagconfig/model/ProviderStatistics.kt

```diff
@@ -0,0 +1,135 @@
+package io.github.inductiveautomation.kindling.idb.tagconfig.model
+
+import io.github.inductiveautomation.kindling.utils.splitCamelCase
+import java.util.Locale
+import kotlin.reflect.KProperty
+
+@Suppress("MemberVisibilityCanBePrivate")
+class ProviderStatistics {
+    val orphanedTags = ListStatistic<Node>("orphanedTags")
+
+    val all = listOf(
```

Comment:
Maybe just use `mapOf` here directly, and read the map's `values` where you refer to `all `currently